lib/skate/settings/ladder_direction.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Skate.Settings.LadderDirection do
+  use Ecto.Type
```

Comment:
This is as much related to the DB as the schemas are, so it seems inconsistent that the schemas are in `db/` while this isn't.
I see 3 options:
1. Move the `Ecto.Type`s into `db/`
2. rename `db/` to `schema/`
3. Leave it, it's fine.